sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -53,12 +54,16 @@ public void Ctor_Uri()
             var accountKey = Convert.ToBase64String(new byte[] { 0, 1, 2, 3, 4, 5 });
             var blobEndpoint = new Uri("http://127.0.0.1/" + accountName);
             var credentials = new StorageSharedKeyCredential(accountName, accountKey);
+            var tokenCredentials = new SharedTokenCacheCredential();
```

Comment:
Any reason to not use `DefaultAzureCredential` since we're not actually making any requests?